.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Please add a header comment describing in general what this shell script does.
Also add some comments below to describe the different major sections of code.